api/Document.json

```diff
@@ -2611,29 +2611,16 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/createTouch",
           "support": {
-            "chrome": [
-              {
-                "version_added": "22",
-                "version_removed": "66"
-              },
-              {
-                "version_added": "59",
-                "notes": "All parameters optional"
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "25",
-                "version_removed": "66"
-              },
-              {
-                "version_added": "59",
-                "notes": "All parameters optional"
-              }
-            ],
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": "25",
+              "version_removed": "68",
```

Comment:
Removal in 68 confirmed by https://storage.googleapis.com/chromium-find-releases-static/112.html#1129d0d679e25123091792887b463ac9c2606297